rebasehelper/output_tools/text_output_tool.py

```diff
@@ -120,40 +148,35 @@ def print_summary(cls, path, results):
         try:
             LoggerHelper.add_file_handler(logger_report, path)
         except (OSError, IOError):
-            raise RebaseHelperError("Can not create results file '%s'" % path)
+            raise RebaseHelperError("Can not create results file '{}'".format(path))
 
         cls.results_store = results
 
         cls.print_success_message()
-        type_pkgs = ['old', 'new']
+        logger_report.info("Rebase helper results are located in %s", os.path.dirname(path))
```

Comment:
What is `Rebase helper`? We should refer to it as `rebase-helper`. But I think something like this would be better here:
`All result files are stored in %s`.